README.md

```diff
@@ -164,6 +164,43 @@ const timber = await User.findOneBy({
 await timber.remove()
 ```
 
+## Null and Undefined Handling
```

Comment:
I'm not entirely convinced that this section belongs in the project README. Explaining it in the docs should be enough?

Reply:
100% Will remove 🙏 
Thanks for the feedback!